Description: Upstream changes introduced in version 3.14-3
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 hmake (3.14-3) UNRELEASED; urgency=low
 .
   [ Joachim Breitner ]
   * Section: haskell
 .
   [ Marco Silva ]
   * Use ghc instead of ghc6
 .
   [ Joachim Breitner ]
   * Fix FTBFS: Compat.hs:47:55: Not in scope: `isAlphanum' (Closes: #628308)
 .
 The person named in the Author field signed this changelog entry.
Author: Joachim Breitner <nomeata@debian.org>
Bug-Debian: http://bugs.debian.org/628308

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- hmake-3.14.orig/src/hmake/Makefile
+++ hmake-3.14/src/hmake/Makefile
@@ -48,7 +48,7 @@ ifeq "nhc" "$(BUILDCOMP)"
 HEAP = -H4M
 endif
 ifeq "ghc" "$(BUILDCOMP)"
-HFLAGS += $(shell ${LOCAL}fixghc ${GHCSYM} -package lang)
+HFLAGS += $(shell ${LOCAL}fixghc ${GHCSYM} -package lang -package haskell98 )
 endif
 ifeq "hbc" "$(BUILDCOMP)"
 all: hbc
--- hmake-3.14.orig/src/interpreter/Makefile
+++ hmake-3.14/src/interpreter/Makefile
@@ -11,7 +11,7 @@ CHFILES = $(patsubst %.hs, ../hmake/%.$C

 ifeq "${BUILDCOMP}" "ghc"
 HFLAGS = $(shell $(LOCAL)fixghc $(GHCSYM) \
-      -package lang -package util -package base -package editline $(READLINE) )
+      -package lang -package util -package base -package editline -package haskell98 $(READLINE) )
 export HFLAGS
 endif
 ifeq "${BUILDCOMP}" "hbc"
